specification/cognitiveservices/data-plane/ComputerVision/preview/v3.2-preview/Ocr.json

```diff
@@ -347,22 +347,36 @@
       }
     },
     "Appearance": {
-      "description": "An object representing the style and styleConfidence.",
+      "description": "An object representing the appearance of the text line.",
       "type": "object",
       "required": [
-        "style",
-        "styleConfidence"
+        "style"
       ],
       "properties": {
         "style": {
-          "description": "The text line style.",
+          "description": "An object representing the Style of the text line.",
+          "type": "object",
+          "$ref": "#/definitions/Style"
+        }
+      }
+    },
+    "Style": {
+      "description": "An object representing the style of the text line.",
+      "type": "object",
+      "required": [
+        "name",
+        "confidence"
+      ],
+      "properties": {
+        "name": {
+          "description": "The text line style name.",
```

Comment:
Recommend listing the currently supported values in the description with indication that additional styles may be added in the future.

Reply:
Add options to description